src/Implement/RequestHandler.cpp

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "RequestHandler.h"
+
+#include <netinet/in.h>
+#include <sys/socket.h>
+#include <sys/types.h>
+#include <unistd.h>
+
+#include <string>
+
+std::unordered_map<std::string, APIHandler *> RequestHandler::apis;
+
+RequestHandler::RequestHandler() { this->read_size = 4096; }
+
+void RequestHandler::SetFd(const int &file_descriptor) {
+  this->client_fd = file_descriptor;
+}
+
+int RequestHandler::ReadHeader(std::string &header,
```

Comment:
If you are overwriting `header`, consider using a pointer.

Reply:
Done